docset/winserver2025-ps/defender/Set-MpPreference.md

```diff
@@ -1537,7 +1537,9 @@ Accept wildcard characters: False
 ```
 
 ### -ScanScheduleOffset
-Configures the number of minutes after midnight to perform a scheduled scan. The time on the endpoint is used to determine the local time. If you enable this setting, a scheduled scan will run at the time specified. If you disable or don’t enable this setting, a scheduled scan runs at the default time of two hours (120 minutes) after midnight.
+Configures the number of minutes after midnight to perform a scheduled scan. The time on the endpoint is used to determine the local time. If you enable this setting, a scheduled scan will run at the time specified. If you disable or don’t enable this setting, a scheduled scan runs at the default time of two hours (120 minutes) after midnight. 
+
+_Note: Recommended to use ScanScheduleOffset for a more intuitive appraoch when setting the time for the Weekly Scheduled Scan (alongside ScanParameters and ScanScheduleDay). If both ScanScheduleOffset and ScanScheduleTime are set, whichever configuration was edited more recently will be used to determine the time to execute the scan at._
```

Comment:
```suggestion
> [!TIP]
> We recommend using the **ScanScheduleOffset** parameter instead of the **ScanScheduleTime** parameter because it's a more intuitive way to set the time for the Weekly Scheduled Scan (in addition to the **ScanParameters** and **ScanScheduleDay** parameters). If both **ScanScheduleOffset** and **ScanScheduleTime** are set, whichever configuration was edited more recently is used to determine the time to execute the scan.
```